src/services/lnd/schema.ts

```diff
@@ -4,26 +4,11 @@ import * as mongoose from "mongoose"
 const Schema = mongoose.Schema
 
 const confirmedDetailsSchema = new Schema<LnPaymentConfirmedDetails>({
-  confirmedAt: {
-    type: Date,
-    required: true,
-  },
-  destination: {
-    type: String,
-    required: true,
-  },
-  revealedPreImage: {
-    type: String,
-    required: true,
-  },
-  roundedUpFee: {
-    type: Number,
-    required: true,
-  },
-  milliSatsFee: {
-    type: Number,
-    required: true,
-  },
+  confirmedAt: Date,
+  destination: String,
+  revealedPreImage: String,
+  roundedUpFee: Number,
+  milliSatsFee: Number,
```

Comment:
shouldn't this match non optional field in https://github.com/alexbosworth/ln-service#getpayments ? (I mean for the `required` flag)